domains/itzliam

```diff
@@ -0,0 +1,9 @@
+{
+"owner": {
+"username": "liamgaming23",
+"email": "liamstickman123@gmail.com"
+},
+"record": {
+"CNAME": "liamgaming23.github.io/liam.github.io"
```

Comment:
Please remove the `/` and everything after it.

Reply:
did it